projects/showcase/test/specs/alphanumeric-layout.spec.ts

```diff
@@ -101,15 +93,17 @@ describe("AlphanumericLayout", () => {
     it("Click again on 'Caps Lock' to return to lowercase layout", async () => {
         await VirtualKeyboard.get().clickCapsLock();
 
-        await ReportUtility.addExpectedResult("Virtual keyboard is shown again with all lowercase letters and numbers", async() => {
-            expect(await VirtualKeyboard.get().isPresent()).toBeTruthy();
-            expect(await VirtualKeyboard.get().getRowCount()).toEqual(5);
-            expect(await VirtualKeyboard.get().getRowKeys(0)).toEqual([      '`', '1', '2', '3', '4', '5', '6', '7', '8', '9', '0', '-', '=', '{bksp}']);
-            expect(await VirtualKeyboard.get().getRowKeys(1)).toEqual([  '{tab}', 'q', 'w', 'e', 'r', 't', 'y', 'u', 'i', 'o', 'p', '[', ']', '\\']);
-            expect(await VirtualKeyboard.get().getRowKeys(2)).toEqual([ '{lock}', 'a', 's', 'd', 'f', 'g', 'h', 'j', 'k', 'l', ';', '\'', '{enter}']);
-            expect(await VirtualKeyboard.get().getRowKeys(3)).toEqual(['{shift}', 'z', 'x', 'c', 'v', 'b', 'n', 'm', ',', '.', '/', '{shift}']);
-            expect(await VirtualKeyboard.get().getRowKeys(4)).toEqual(['{space}']);
-        });
+        await expectAlphanumericLowercaseLayout();
+    });
+
+    it("Click on 'Shift' virtual keyboard key to change layout to uppercase for only once key", async () => {
+        await VirtualKeyboard.get().clickShift();
+
+        await expectAlphanumericUppercaseLayout();
+
```

Comment:
The code is OK, but to keep the actions vs expected results separation, the click on the S key and the later expectation should be placed on a separated "it".